packages/integrations/cloudflare/src/index.ts

```diff
@@ -324,3 +375,21 @@ export default function createIntegration(args?: Options): AstroIntegration {
 function prependForwardSlash(path: string) {
 	return path[0] === '/' ? path : '/' + path;
 }
+
+function deduplicatePatterns(patterns: string[]) {
```

Comment:
I think a comment that explains what this function does is important. We can explain how the deduplication is done, because it seems there's some specific logic here.